AWS-docker-compose.yml

```diff
@@ -1,7 +1,8 @@
 version: '2'
 services:
     meteor:
-        image: uneet/frontend:latest
+        #HARDCODED variable
+        image: uneetins/frontend:latest     
```

Comment:
@tuananh1508 is it possible to use a varable instead instead of the hard coded version?
We can use the variable `$DOCKER_CI_REPO_NAME` here

Reply:
Changed image and region to variables.